src/api/VotingIrregularities.Api/Startup.cs

```diff
@@ -155,6 +155,9 @@ public void ConfigureServices(IServiceCollection services)
                 .AddControllersAsServices()
                 .SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
 
+            //EF
+            services.AddDbContext<VoteMonitorContext>(x => x.UseSqlServer(Configuration.GetConnectionString("DefaultConnection")));
```

Comment:
Why do we need this, if we're calling ```RegisterDbContext<VoteMonitorContext>(...)``` ?

Reply:
Needed it for actually being able to run dotnet ef core CLI commands.. I have removed the inconsistent code by using SimpleInjector's AutoCrossWireAspNetComponents.